newlib/libc/machine/loongarch/asm.h

```diff
@@ -0,0 +1,50 @@
+/* Copyright (c) 2024 Jiaxun Yang <jiaxun.yang@flygoat.com>
+
+   This copyrighted material is made available to anyone wishing to use,
```

Comment:
Could you replace this license text with the text used in newlib/libc/machine/loongarch/machine/fenv.h? 'The FreeBSD license' isn't well defined.